opensearch/services/datatypes/datatypes.php

```diff
@@ -0,0 +1,430 @@
+		if (isset($result['sourceStandard'])) {
+			$html .= "<br /><span>Source Standard: ";
+			$sourceStandard_arr = explode(",", $result['sourceStandard']);
+			$sourceStandard_label_arr = explode(",", $result['sourceStandard_label']);
+			$sourceStandard_markup = array();
+			foreach ($group_arr as $i => $sourceStandard) {
+				array_push($sourceStandard_markup, "<a target='_blank' href=\"" . $sourceStandard . "\">" . $sourceStandard_label_arr[$i] . "</a>");
+			}
+			$html .= implode('; ', $sourceStandard_markup);
+			$html .= "</span>";
+		}
+
+		// Parameters
+		$parameters = $this->getParametersByDatatype($result['datatype']);
+		if (count($parameters) > 0) {
+			$html .= "<br /><span>Parameters: ";
+			if (count($parameters) > 0) {
+				$parameters_markup = array();
+				foreach ($parameters as $i => $parameter) {	
+					array_push($parameters_markup, "<a target='_blank' href=\"" . $parameter['uri'] . "\">" . $parameter['label'] . "</a>", "<br>" . $parameter['hasUnit']);
+				}
+				$html .= implode('; ', $parameters_markup);
+			}
+			$html .= "</span>";
+		}
+
+		// Subject Area
+		$subjectAreas = $this->getSubjectAreaByDatatype($result['datatype']);
+		if(count($subjectAreas) > 0){
+			$html .= "<br /><span>Subject Area: ";
+			$subjectAreas_markup = array();
+				foreach($subjectAreas as $key => $subjectArea){
+					array_push($subjectAreas_markup, $subjectArea['concept']);
+				}
+			$html .= implode('; ', $subjectAreas_markup);
+			$html .= "</span>";
+		}
+
+		// Datasets
+		$datasets = $this->getDatasetsByDatatype($result['datatype']);
+		if(count($datasets) > 0){
+			$html .= "<br /><span>Datasets: ";
+			$datasets_markup = array();
+				foreach($datasets as $key => $dataset){
+					array_push($datasets_markup, "<a target='_blank' href=\"" . $dataset['uri'] . "\">" . $dataset['label'] . "</a>");
+				}
+			$html .= implode('; ', $datasets_markup);
+			$html .= "</span>";
+		}
+
+		// access
+		if (isset($result['access'])) {
+			$html .= "<br /><span>Access restriction: " . $result['access'] . "</span>";
+		}
+
+		$html .= "</div>";
+		return $html;
+	}
+	
+	/**
+	* Return SPARQL query header component
+	* @param string $type search type (e.g. 'parameters', 'authors', 'keywords')
+	* @return string query header component (e.g. 'SELECT ?id ?label')
+	*/
+	public function getQueryHeader($type) {
+	
+		$header = "";
+		switch($type) {
+			case "datatypes":
+				$header .= "?datatype ?dco_id ?label ?year ?parameterLabel ?access ";
+				$header .= '(GROUP_CONCAT(DISTINCT ?creator ; SEPARATOR=",") AS ?creator) ';
+				$header .= '(GROUP_CONCAT(DISTINCT ?creator_label ; SEPARATOR=",") AS ?creator_label) ';
+				$header .= '(GROUP_CONCAT(DISTINCT ?sourceStandard ; SEPARATOR=",") AS ?sourceStandard) ';
+                $header .= '(GROUP_CONCAT(DISTINCT ?sourceStandard_label ; SEPARATOR=",") AS ?sourceStandard_label) ';
+				break;
+			case "count":
+				$header .= "(count(DISTINCT ?datatype) AS ?count)";
+				break;
+			default:
```

Comment:
Add a new case statement for when the query parameter is "creationYear".  In this case we are not return ?id and ?label but just ?year.  You will likewise need to add a case for "creationYear" in the getQueryFooter function.

```
case "creationYear":
    $header .= '?year' ;
    break;
```